Python/libraries/recognizers-date-time/recognizers_date_time/date_time/english/holiday_parser_config.py

```diff
@@ -0,0 +1,208 @@
+from typing import List, Dict, Callable
+import re
+from datetime import datetime
+
+from recognizers_text.utilities import RegExpUtility
+from ..utilities import DateUtils
+from ..base_holiday import BaseHolidayParserConfiguration
+from ...resources.english_date_time import EnglishDateTime
+
+class EnglishHolidayParserConfiguration(BaseHolidayParserConfiguration):
+    @property
+    def holiday_names(self) -> Dict[str, List[str]]:
+        return self._holiday_names
+
+    @property
+    def holiday_regex_list(self) -> List[str]:
+        return self._holiday_regexes
+
+    @property
+    def holiday_func_dictionary(self) -> Dict[str, Callable[[int], datetime]]:
+        return self._holiday_func_dictionary
+
+    def get_swift_year(self, text: str) -> int:
+        trimmed_text = text.strip().lower()
+        swift = -10
+        if trimmed_text.startswith('next'):
```

Comment:
Ideally these strings should also come from YAML.

Reply:
This change will affect other platforms, it should be done in other pull-request